### PR TITLE
Tooltips should dispatch a state change when click on close element

### PR DESCRIPTION
**Pull request by calderon**
### What is the goal?

Dispatch a state change when click on close element

- [x] It passses the `jt-frontend` lint commands (`--lint-js`, `--lint-less`, `--lint-coffee`, ...)
- [x] The `README.md` has been updated accordingly

### How is being implemented?

Adding a call to close method from controller

### Reviewers

@jobandtalent/frontend

### CODE REVIEW

File: src/component/tooltip/tooltip-view.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ class TooltipView extends BaseView {
 
         if (clickedElement.className.indexOf(closeElementClass) !== -1 ||
             clickedElement.parentElement.className.indexOf(closeElementClass) !== -1) {
+          self._controller.close();
```
Comment (tomasperezv) [inline src/component/tooltip/tooltip-view.js]: A bit strange to have 2 close actions here, no?

Comment (scumah) [inline src/component/tooltip/tooltip-view.js]: Yep, you could maybe add it in the `close` method

Comment (calderon) [inline src/component/tooltip/tooltip-view.js]: Not quite sure how to handle this. Current close method is what have to be done when closing (set display as none). It's what should happend when we call to close method from here:

```var tooltip = new Tooltip({...});
tooltip.close()````

Am I correct?

But _initEvents handle the click event, a thing totally different of close behaviour. Not sure if this has some logic for you :-D

Comment (calderon) [inline src/component/tooltip/tooltip-view.js]: @tomasperezv maybe remove self.close() to controller method and call it like this._view.close()? It has sense?

Comment (calderon) [inline src/component/tooltip/tooltip-view.js]: Yes it is. That's it. Removing `this.close()` and call to controller close does the trick.

**Pull request merged**